README.md

```diff
@@ -1,4 +1,54 @@
-# data-pack
+# data-pack : data packages for Python
 
-Data packages for Python
+Data-pack is a library for querying and downloading data in Python.
 
+The data-pack specification is given below.  Packages can be
+distributed either directly (and queried via URL), or via an index.
+
+## Specification: data-pack
+
+A data pack is a JSON file, stored at a publicly accessible HTTP URL,
+of the following format:
```

Comment:
Publicly accessible and *persistent* (?). 

That is, the Github link is no good, but the figshare one is OK.